source/extensions/stat_sinks/hystrix/hystrix.cc

```diff
@@ -0,0 +1,379 @@
+  addIntToStream("propertyValue_circuitBreakerSleepWindowInMilliseconds", 0, cluster_info);
+  addIntToStream("propertyValue_circuitBreakerErrorThresholdPercentage", 0, cluster_info);
+  addInfoToStream("propertyValue_circuitBreakerForceOpen", "false", cluster_info);
+  addInfoToStream("propertyValue_circuitBreakerForceClosed", "true", cluster_info);
+  addStringToStream("propertyValue_executionIsolationStrategy", "SEMAPHORE", cluster_info);
+  addIntToStream("propertyValue_executionIsolationThreadTimeoutInMilliseconds", 0, cluster_info);
+  addInfoToStream("propertyValue_executionIsolationThreadInterruptOnTimeout", "false",
+                  cluster_info);
+  addIntToStream("propertyValue_executionIsolationSemaphoreMaxConcurrentRequests",
+                 max_concurrent_requests, cluster_info);
+  addIntToStream("propertyValue_fallbackIsolationSemaphoreMaxConcurrentRequests", 0, cluster_info);
+  addInfoToStream("propertyValue_requestCacheEnabled", "false", cluster_info);
+  addInfoToStream("propertyValue_requestLogEnabled", "true", cluster_info);
+  addIntToStream("reportingHosts", reporting_hosts, cluster_info);
+  addIntToStream("propertyValue_metricsRollingStatisticalWindowInMilliseconds", rolling_window_ms,
+                 cluster_info);
+
+  ss << "data: {" << cluster_info.str() << "}" << std::endl << std::endl;
+}
+
+void HystrixSink::addHystrixThreadPool(absl::string_view cluster_name, uint64_t queue_size,
+                                       uint64_t reporting_hosts, uint64_t rolling_window_ms,
+                                       std::stringstream& ss) {
+  std::stringstream cluster_info;
+
+  addIntToStream("currentPoolSize", 0, cluster_info);
+  addIntToStream("rollingMaxActiveThreads", 0, cluster_info);
+  addIntToStream("currentActiveCount", 0, cluster_info);
+  addIntToStream("currentCompletedTaskCount", 0, cluster_info);
+  addIntToStream("propertyValue_queueSizeRejectionThreshold", queue_size, cluster_info);
+  addStringToStream("type", "HystrixThreadPool", cluster_info);
+  addIntToStream("reportingHosts", reporting_hosts, cluster_info);
+  addIntToStream("propertyValue_metricsRollingStatisticalWindowInMilliseconds", rolling_window_ms,
+                 cluster_info);
+  addStringToStream("name", cluster_name, cluster_info);
+  addIntToStream("currentLargestPoolSize", 0, cluster_info);
+  addIntToStream("currentCorePoolSize", 0, cluster_info);
+  addIntToStream("currentQueueSize", 0, cluster_info);
+  addIntToStream("currentTaskCount", 0, cluster_info);
+  addIntToStream("rollingCountThreadsExecuted", 0, cluster_info);
+  addIntToStream("currentMaximumPoolSize", 0, cluster_info);
+
+  ss << "data: {" << cluster_info.str() << "}" << std::endl << std::endl;
+}
+
+void HystrixSink::addClusterStatsToStream(absl::string_view cluster_name,
+                                          uint64_t max_concurrent_requests,
+                                          uint64_t reporting_hosts, uint64_t rolling_window_ms,
```

Comment:
nit: Sorry for being indecisive on this, but could we instead just pass the rolling_window_ms around as type `std::chrono::milliseconds` rather than a `uint64_t`?  This means that if the flush interval that you're accessing starts being represented as `std::chrono::microseconds` or `std::chrono::seconds`, you'll see a compile-time error.  I know this is unlikely, but we've had problems with outdated `.count()` interpretations in the past.